packages/nest/README.md

```diff
@@ -25,11 +25,43 @@
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
 for feature flagging that works with your favorite feature flag management tool.
 
+<!-- x-hide-in-docs-end -->
+
 🧪 This SDK is experimental.
 
-#### Here's a basic example of how to use the OpenFeature NestJS API with `InMemoryProvider`.
+## Overview
+
+The OpenFeature NestJS SDK is a package that provides a NestJS wrapper for the [OpenFeature Server SDK](). It's main capabilities are:
+- Provide a NestJS global module to simplify OpenFeature configuration and usage within NestJS;
+- Supply custom parameter decorators (for controllers), that inject observables with feature flags resolution details;
+- Inject context for flag evaluation seamlessly using NestJS interceptors
+- Provide decorators for OpenFeature Client injection into NestJS services and controllers
+- TODO: map other features
+
+## 🚀 Quick start
+
+### Requirements
+
+- Node.js version 16+
+- NestJS
+
+### Install
+
+#### npm
+
+```sh
+npm install --save @openfeature/nestjs-sdk
+```
+
+#### yarn
+
+```sh
+yarn add @openfeature/nestjs-sdk
```

Comment:
Yarn unfortunately doesn't automatically install peer dependencies. Please make sure to list any extra dependencies that are required.

Reply:
Could be something like this: https://github.com/open-feature/js-sdk-contrib/tree/main/libs/providers/config-cat#installation

Reply:
@luizgribeiro What do you think?

Reply:
It's a nice alternative. I'll rewrite this